cellxgene_schema_cli/cellxgene_schema/atac_seq.py

```diff
@@ -209,10 +209,16 @@ def process_fragment(
         else:
             logger.info("Fragment and Anndata file are valid")
 
+        organism_ontology_term_id = None
+        with h5py.File(anndata_file) as f:
+            organism_ontology_term_id = ad.io.read_elem(f["uns"]).get("organism_ontology_term_id")
+
```

Comment:
this gets called in a couple of the functions called by validate_anndata_with_fragment. probably worth fetching once and passing as an arg, I think